tests/integration/_cases/sourcemaps/sourcemaps-upload-some-debugids.trycmd

```diff
@@ -25,8 +25,7 @@ $ sentry-cli sourcemaps upload tests/integration/_fixtures/upload_some_debugids
 
 Source Map Upload Report
   Scripts
-    ~/server/chunks/flight-server-css-manifest.js
-  Minified Scripts
+    ~/server/chunks/flight-server-css-manifest.js (no sourcemap ref)
```

Comment:
One day we can maybe reword `no sourcemap ref` to something more clear. Eg `no source map reference` or `no source map found.`

Reply:
Good idea, I already opened an issue: #1964